src/bundle/Resources/views/content/create/create.html.twig

```diff
@@ -8,23 +8,23 @@
 
 {% block details %}
     <div class="container mt-5 px-5">
-        <h2 class="ez-content-item-status">{{ 'creating_in_language'|trans({'%contentTypeName%': contentType.name, '%languageName%': language.name})|desc('Creating - %contentTypeName% in %languageName%') }}</h2>
+        <h2 class="ez-content-item-status">{{ 'creating_in_language'|trans({'%contentTypeName%': content_type.name, '%languageName%': language.name})|desc('Creating - %contentTypeName% in %languageName%') }}</h2>
         <h1>
-            <svg class="ez-icon ez-icon-{{ contentType.identifier }}">
-                <use xlink:href="{{ ez_content_type_icon(contentType.identifier) }}"></use>
+            <svg class="ez-icon ez-icon-{{ content_type.identifier }}">
+                <use xlink:href="{{ ez_content_type_icon(content_type.identifier) }}"></use>
             </svg>
-            {{ 'new_content_item'|trans({'%contentType%': contentType.name})|desc('New %contentType%') }}
+            {{ 'new_content_item'|trans({'%contentType%': content_type.name})|desc('New %contentType%') }}
         </h1>
 
         <div class="small">
-            {{ contentType.name }} / Parent Location ID: {{ parentLocation.id }}
+            {{ content_type.name }} / Parent Location ID: {{ location.id }}
```

Comment:
I am against changing `parent location` to `location` in this context. Below we are good with `parent_location`

Reply:
https://github.com/ezsystems/repository-forms/blob/89d28cdea7e1639193a92c6eee929172964381d3/lib/Content/View/Builder/ContentCreateViewBuilder.php#L121 

there was a @todo to rename it

Reply:
@Nattfarinn, I think you added this @todo comment. Also @webhdx share your opinion, please.